### ISSUE TITLE
Help to analise transportation graph

### ISSUE THREAD

**Issue by fabiocorreacordeiro** [datasciencebr/serenata-de-amor#215]
Hi guys,
I am working in local transportation subquotas ('Taxi, toll and parking', 'Automotive vehicle renting or charter' and 'Fuels and lubricants'). When I sum all the expenses per month I found the graph below.
![ps4 fig 1](https://cloud.githubusercontent.com/assets/26165406/25273364/2a117a60-2694-11e7-95ad-cc636efee3d2.png)
Anyone know to explain why the expenses increase after 2014?

The second question was about the expenses of each congressman. In the follows graphs, I plot a bar with the median (center dot), quantile 95% and 5% (upper and down limit of each bar). The first graph is with all congressman, the other two I divided by mean greater and less than R$5,000 per month for a better visualization.
![ps4 fig 3](https://cloud.githubusercontent.com/assets/26165406/25273609/2e034d3c-2695-11e7-8189-562e8f4316aa.png)
![ps4 fig 4](https://cloud.githubusercontent.com/assets/26165406/25273611/2e0babbc-2695-11e7-99f2-6567a405d602.png)
![ps4 fig 5](https://cloud.githubusercontent.com/assets/26165406/25273610/2e0a61da-2695-11e7-9952-e35f4721946d.png)
My question is why a lot of monthly expenses (not all) was limited in R$4,500? There is some rule limited in this value?

**Comment by cuducos** [comment]
TBH IDK but there are some [monthly limits](https://github.com/datasciencebr/serenata-de-amor/blob/master/CONTRIBUTING.md#more-about-the-quota-for-exercising-parliamentary-activity-ceap), maybe this value is a kind of median of these limits… I think that a notebook sharing this analysis concluding with some questions is a valid contribution.

**Comment by fabiocorreacordeiro** [comment]
I am working in a notebook, but I did these analyses to my R Studio class. Do you think I need to change by Python Language?

I am studying better the CEAP legislation and its change since 2009. In the beginning, there was only limit for 'Fuels and lubricants', R$4,500. In December of 2013, the total CEAP value of each state was an 7% increase. Apparently, it increase affected the expenses with local transportation.
In Feb/2015 it was established limits to 'Taxi, toll and parking' (R$2,700) and 'Automotive vehicle renting or charter' (R$10,900) and in Sep/2015 the limit of 'Fuels and lubricants' jump to R$6,000.

So, probably the median that we can observe in the graphs was the old limit of R$4,500 in 'Fuels and lubricants'. Another curiosity is that when the limits were established the expenses increase. Probably, after put limits in the legislation, the congressman fell free to use R$10,900 to rent a car and use R$6,000 to fill the tank.

**Comment by cuducos** [comment]
This is very interesting, @fabiocorreacordeiro — many thanks.

Even if this isn't illegal _per se_ I think this is the kind of news that is worth it for the media. On the one hand we can call into question the legality of certain expenses, but on the other hand we can always call into question the morality — and comments such as this one are totally valid in this second sense:

> Probably, after put limits in the legislation, the congressman fell free to use R$10,900 to rent a car and use R$6,000 to fill the tank.

I'd advise you to advance on this notebook and analysis and later — if you agree — we can use our network to put this story forward in the media. What do you think?

**Comment by fabiocorreacordeiro** [comment]
Sure I agree!
I was thinking exactly the same thing about legality and morality. Even some expense was legal, but it was immoral, we can be able to deliver the complaint. 
I will continue my work and any news I let you know.

**Comment by jtemporal** [comment]
@fabiocorreacordeiro I would love to see this notebook of yours, whereas Rosie is strictly Python, you have all the freedom to make a PR with notebook written in R, as long as you make us (everyone contributing for the project) know what steps we need to make to reproduce the results ;)

I would recommend that you take a look at #130 it's a little long but it's worth the read on the matter of "how to contribute in languages other than Python".

The R side of me is looking forward to this PR, great work and valid questioning =)

**Comment by fabiocorreacordeiro** [comment]
@jtemporal, thanks for the help, but I am already writing a notebook in Python with the main topics.
One thing that I discover in this review was why transportation had a jump in 2014. The subquota about rent cars and boat was one before 2014 and was split in two differents subquotas. In my analysis, I was missing the reimbursements before 2014.

**Comment by fabiocorreacordeiro** [comment]
Hi guys,
Yesterday I finished my notebook, but I not sure if the files were merged correctly. Could you check to me?
Anyway, I attached here the three file, py, html and ipynb.
I want to hear you about the suspicious reimbursements and what could be the next steps.

[2017-04-29-fabiocorreacordeiro-Local-Transportation.zip](https://github.com/datasciencebr/serenata-de-amor/files/965937/2017-04-29-fabiocorreacordeiro-Local-Transportation.zip)

**Comment by fabiocorreacordeiro** [comment]
@cuducos and @jtemporal ,
I am not sure if my notebook was merged, could you confirm to me?
If no, let me know.
Thanks

**Comment by jtemporal** [comment]
@fabiocorreacordeiro The notebook in the zip file is part of #203 right? if so it was not merged yet.

**Comment by fabiocorreacordeiro** [comment]
@jtemporal No, I wrote this notebook in "serenata-de-amor/develop" in my forked directory. I created a commitment, but the Github merged this commitment without a new pull request.
Or I am doing something wrong :)

**Comment by fabiocorreacordeiro** [comment]
@jtemporal and @cuducos  Finally I created the Pull Request #228 
I was doing something wrong ... ahahah

**Comment by jtemporal** [comment]
@fabiocorreacordeiro PR commented ;)

**Comment by cuducos** [comment]
@fabiocorreacordeiro as far as I could get you authored three of our opened PRs: #203, #206 and #228.

In #206 we are a little behind and a code review is still pending (@anaschwendler or @jtemporal — are you able to check it?).

#203 and #228 seems to have code that do not belong to them (@irio's commits 0a28212 and 6f2557c). You have to clean up that to avoid conflicts when we merge.

From our previous conversation in #203 I'm not sure if you're done with it an if I someone from our side can go through a new code review.

Finally regarding #228 @jtemporal offered some code review — we cannot merge until you address her concerns.

Is that helpful?

**Comment by fabiocorreacordeiro** [comment]
@cuducos , @jtemporal and @anaschwendler ,
these three PRs are about the same theme with distinct maturity. Do you think is a good idea to consolidate them in one new PR and delete the PRs #203, #206 and #228?

**Comment by cuducos** [comment]
I don't mind having three PR. It actually makes it easy for us to code review (I'm reviewing one of them, @jtemporal is reviewing another one, for example). The thing is that we cannot merge if you don't respond to our feedback — @jtemporal and I are still looking forward to hear from you in #203 and #228 ; )

**Comment by cuducos** [comment]
Closed by #232
